algoliasearch.gemspec

```diff
@@ -65,16 +65,16 @@ Gem::Specification.new do |s|
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
       s.add_runtime_dependency     'httpclient', '~> 2.8', '>= 2.8.3'
-      s.add_runtime_dependency     'json',       '>= 1.5.1'
+      s.add_runtime_dependency     'json',       '>= 1.5.1', '< 2.3'
       s.add_development_dependency 'travis',     '~> 0'
       s.add_development_dependency 'rake',       '~> 0'
       s.add_development_dependency 'rdoc',       '~> 0'
     else
       s.add_dependency             'httpclient', '~> 2.8', '>= 2.8.3'
-      s.add_dependency             'json',       '>= 1.5.1'
+      s.add_dependency             'json',       '>= 1.5.1', '< 2.3'
```

Comment:
Can this change effect current customers?

Reply:
in this case and the next one, it targets older versions of Ruby which would have the same dependency problem so we can block the version